installer.md

```diff
@@ -3,28 +3,36 @@ layout: default
 ---
 ![R and VS](./media/r.plus.vs.png)
 
-## Thank you for installing RTVS Public Preview!
+## Thank you for installing RTVS!
 
 ![RTVS Screenshot](media\installer_screenshot.png)
 
 ### About this release
 
-THANK YOU for checking out this early version for RTVS! Please check it out and
-let us know what you think. This release is meant for evaluation purposes only
-and not for production use.
+Thank you for installing R Tools for Visual Studio! Please check it out and let
+us know what you think. 
 
 ### Resources
 
-* For general overview of the features, please see the [documentation.](./index.html)
-* To install RTVS, especially if you are new to Visual Studio, please see [Installation steps.](./installation.html)
-* For a quick overview, please watch this [short video.](https://www.youtube.com/watch?v=KPS0ytrt9SA) 
-* For a list of existing bugs and feature requests please visit our [Github repo](https://github.com/Microsoft/RTVS/issues) 
-* If you'd like to check out the source code, license and contribute, please see the [contribution guide](https://github.com/Microsoft/RTVS/blob/master/LICENSE).
+* For general overview of the features, please see the
+  [documentation.](./index.html)
+* To install RTVS, especially if you are new to Visual Studio, please see
+  [Installation steps.](./installation.html)
+* For a quick overview, please watch this [short
+  video.](https://www.youtube.com/watch?v=KPS0ytrt9SA)
```

Comment:
Good catch. I'll update the video to the latest 0.4 one that we have. I will shoot a brand new one for 1.0